R/LogLogisticOs.R

```diff
@@ -0,0 +1,36 @@
+
+#'  Log-logistic overall survival
+#'
+#' Description - Log-logistic overall survival model object creator
+#'
+#' @export
+LogLogisticOs <- setClass(
+    Class = "LogLogisticOs",
+    contains = "OsModel"
+)
+
+
+#' LogLogisticOs object creator
+#'
+#' Creates a log-logistic overall survival templated object
+#' @rdname LogLogisticOs-class
+#' @importFrom assertthat assert_that
+#' @export
+setMethod(
+    f = "initialize",
+    signature = "LogLogisticOs",
+    definition = function(.Object,
+                          ..., stan = LogLogisticModule()) {
```

Comment:
Do we even need to have this as an argument with default ? Would it not be better to just harcode the `stan = LogLogisticModule()` in line 27?

Reply:
`LogLogisticModule` is the core object containing all the stan code for the overall survival model. The user can either use our code or provide his own code (either by directly typing into R or as a .stan file). It should be as a default option for the case that the user wants to provide his of version of a loglogistic model. 
https://github.com/Genentech/jmpost/blob/1b4df12d55d50e10d79b99667a452db84efcd785/R/OsModel.R#L29-L35

Reply:
Since the pattern is already in use in other places (e.g. exponential longitudinal model branch etc.) we keep this for now as is, potentially revisit in the future.